compiler/src/iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -206,6 +206,11 @@ int getReductionTilingFactor(int64_t dimSize);
 // given Linalg op.
 int64_t getMinElementBitwidth(linalg::LinalgOp linalgOp);
 
+// Getter/setter for marking a loop for unrolling.
+void setLoopUnrollMarker(Operation *op);
+bool getLoopUnrollMarker(Operation *op);
+void removeLoopUnrollMarker(Operation *op);
```

Comment:
I'm not a fan about having markers. :/ Do we move the functions to `MarkerUtils.h`, so we can have all the markers in the same files?

An optional change is that we can replace `kLinalgTransformMarker` with `kTransformMarker` and reuse setMarker/hasMarker methods.

https://github.com/iree-org/iree/blob/0b17c728b13189c909ce8c24ef0e048c01ab4758/compiler/src/iree/compiler/Codegen/Utils/MarkerUtils.h#L117-L122

Reply:
I didn't know about marker utils, that sounds like a good place for this.

With regards to use of markers here, I agree in general, but there is a tradeoff between plumbing through an attribute on the operation itself (upstream discussion etc.) and just letting it be a discardable attr. I initially was going to go for a permanent attr, but then found that this didn't seem to get dropped and went with it (I think people have tried hard to keep discardable attrs on scf.for ops alive :). Also I think it's ok in this case because unrolling is best-effort anyway and doesn't matter for correctness.

Reply:
I liked Jakub's suggestion of returning the attribute and the `__internal_linalg_transform__` stuff kind of confused me, so I moved it over but opted against changing the existing linalg marker code.

Reply:
SGTM, thanks